### PR TITLE
Complete the new UI 

### PR DESCRIPTION
**Pull request by fcebk17**
- delete useless pages
- 資料庫尚未連接完成，正在進行中
- 目前只是將版型建構出來，按鈕皆無法觸發

### CODE REVIEW

**Comment by asas1asas200** [comment]
```shell
$ python app.py
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
Traceback (most recent call last):
  File "/home/zeng/Desktop/ElecTrap_backend/app.py", line 27, in <module>
    def rank():
  File "/home/zeng/Desktop/ElecTrap_backend/venv/lib/python3.10/site-packages/flask/scaffold.py", line 440, in decorator
    self.add_url_rule(rule, endpoint, f, **options)
  File "/home/zeng/Desktop/ElecTrap_backend/venv/lib/python3.10/site-packages/flask/scaffold.py", line 56, in wrapper_func
    return f(self, *args, **kwargs)
  File "/home/zeng/Desktop/ElecTrap_backend/venv/lib/python3.10/site-packages/flask/app.py", line 1083, in add_url_rule
    rule = self.url_rule_class(rule, methods=methods, **options)
  File "/home/zeng/Desktop/ElecTrap_backend/venv/lib/python3.10/site-packages/werkzeug/routing.py", line 703, in __init__
    raise ValueError("urls must start with a leading slash")
ValueError: urls must start with a leading slash
```

File: app.py [hunk 1]
```diff
@@ -18,40 +18,15 @@ def test():
...
-@app.route('/levelmap.html')
-def levelmap():
-    """ Select level map page."""
-    return render_template('levelmap.html')
-
-@app.route('/master.html')
-def master_mode():
-    """ Select Master mode page."""
-    return render_template('master.html')
+@app.route('rank.html')
```
Comment (asas1asas200) [inline app.py]: Leading slash is needed.

File: app.py [hunk 2]
```diff
@@ -18,40 +18,15 @@ def test():
...
-def bodymode():
-    """Select body mode page."""
-    return render_template('bodySelect.html')
-
-@app.route('/foot_levelmap.html')
-def foot_levelmap():
-    """ Select Foot Level map page."""
-    return render_template('foot_levelmap.html')
-
 @app.route('/gamemode.html')
```
Comment (asas1asas200) [inline app.py]: The `.html` suffix is not needed

**Review by asas1asas200 — CHANGES_REQUESTED** [review]
Remove the `html` suffix and add a slash on rank.

**Comment by asas1asas200** [comment]
Don't commit bootstrap files.

File: app.py (hunk 1 above)
Comment (fcebk17) [reply app.py]: Complete

File: app.py (hunk 2 above)
Comment (fcebk17) [reply app.py]: fixed yet

**Pull request merged**